src/lbcamden/components/search/search.yaml

```diff
@@ -57,4 +74,17 @@ examples:
   - name: labelText
     hidden: true
     data:
-      labelText: 'Search test'
\ No newline at end of file
+      labelText: 'Search test'
+  - name: additional parameters
+    hidden: true
```

Comment:
@oluoluoxenfree As discussed we think these examples should not be "hidden", so please remove this and add descriptions for the examples

Reply:
Also applies to the examples below.